### ISSUE TITLE
loss is nan

### ISSUE THREAD

**Issue by ke1ulu** [tsingqguo/misf#26]
Hello, I think your results are great, but I have the following problem when trying to train with place2 dataset, can you tell me the reason?
![image](https://github.com/tsingqguo/misf/assets/141639005/2ec0f6fa-4cdd-46dd-a5b4-53b1699c8ee3)

**Comment by Li-Xiaoguang** [comment]
I did not find this issue during my training. Could you please tell me the configuration you used, such as the learning rate, and batch size? Or any other setup which is different from the original configuration.
